### DISCUSSION TITLE
ability to backup hive keys for backup

### DISCUSSION THREAD

**Discussion by lcia-projects** [telekom-security/tpotce#1120] [Q&A] [answered]
hi, looks like my hive server has died. is it possible to backup and restore the ssh keys (i think they are ssh) so i dont have to redeploy my hive nodes? where would these be located?

**Comment by t3chn0m4g3** [comment, marked as answer]
The recommended way is to [redeploy](https://github.com/telekom-security/tpotce/blob/master/README.md#redeploy-hive-sensor) the sensors. You will find a description on key handling in the [Readme](https://github.com/telekom-security/tpotce/blob/master/README.md#distributed-deployment).

**Reply by lcia-projects** [reply]
thank you, that'll do
